src/bindings/python/src/pyopenvino/graph/dict_attribute_visitor.cpp

```diff
@@ -272,7 +272,7 @@ util::DictAttributeSerializer::DictAttributeSerializer(const std::shared_ptr<ov:
 }
 void util::DictAttributeSerializer::on_adapter(const std::string& name, ov::ValueAccessor<void>& adapter) {
     if (m_attributes.contains(name)) {
-        NGRAPH_CHECK(false, "No AttributeVisitor support for accessing attribute named: ", name);
+        OPENVINO_ASSERT("No AttributeVisitor support for accessing attribute named: ", name);
```

Comment:
```suggestion
        OPENVINO_ASSERT(false, "No AttributeVisitor support for accessing attribute named: ", name);
```
Otherwise it don't work because string is true

Reply:
There should be `OPENVINO_THROW`, forgot to change it, so updated